node/rpc/src/lib.rs

```diff
@@ -181,6 +182,7 @@ where
         .post(rpc_http_handler::<DB, B>);
 
     info!("Ready for RPC connections");
+    set_event();
```

Comment:
No need to signal the event here. Do it after the RPC had had a chance to bind to the network port in the daemon.

Reply:
Done.